### PR TITLE
Index Alsos - Remove padding bottom

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #NUMBER

**Overall change:** 
When pulling in the Index Alsos into Simorgh, we realized that there was a lot of spacing beneath top stories with index alsos and the following story, a total of `32px` when it should be a `24px`.

**Code changes:**
- Remove `padding-bottom 8px` from the Index Alsos container

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by andrew-nowak — APPROVED** [review]
lgtm

File: packages/components/psammead-story-promo/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.6.1 | [PR#3216](https://github.com/bbc/psammead/pull/3216) Remove padding-bottom from Index Alsos Wrapper |
```
Comment (ghost) [inline packages/components/psammead-story-promo/CHANGELOG.md]:
```suggestion
| 2.6.1 | [PR#1845](https://github.com/bbc/psammead/pull/1845) Remove padding-bottom from Index Alsos Wrapper |
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: packages/components/psammead-story-promo/CHANGELOG.md (hunk 1 above)
Comment (DenisHdz) [reply packages/components/psammead-story-promo/CHANGELOG.md]: Ups, copied the wrong one 😁

**Review by ghost — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
Looks good 👍

**Pull request merged**